Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.h

```diff
@@ -159,6 +161,8 @@ class RemoteLayerTreeEventDispatcher : public ThreadSafeRefCounted<RemoteLayerTr
     std::optional<DisplayLinkObserverID> m_displayRefreshObserverID;
     PAL::HysteresisActivity m_wheelEventActivityHysteresis;
 
+    int m_ScrollingPerformanceTestSignpostEndIntervals = 0; // Used to prevent multiple end signpost calls
```

Comment:
I think this is accessed from different threads, so needs to be `std::atomic<int>`

The name is rather unclear. What does it really represent? Is it counting the end of `ScrollingPerformanceTestScrollInterval` or `ScrollingPerformanceTestMomentumInterval`?